packages/ra-ui-materialui/src/list/filter/FilterLiveSearch.tsx

```diff
@@ -26,6 +26,7 @@ export const FilterLiveSearch = memo(
     (props: {
         source?: string;
         sx?: SxProps;
+        fullWidth?: boolean;
         variant?: 'filled' | 'outlined';
```

Comment:
Would you mind extracting this into a `FilterLiveSearchProps` interface and export it?

Reply:
Done my dear